src/styles/utils/generators/TooltipStyleUtils/index.native.ts

```diff
@@ -0,0 +1,178 @@
+import {Animated, StyleSheet} from 'react-native';
```

Comment:
File-scoped comment.

Again, the amount of duplication between index.tsx and index.native.tsx is unacceptable.

Reply:
This thread is open

Reply:
Well, we did, but do you suggest that we should put a "done" checkbox here? ✅

If you use `isWeb`, I'll comment: "Please use platform-specific files"

If you implement platform-specific files with duplicated code, I'll comment "Please de-duplicate"

We're currently at "Please de-duplicate" 🤷‍♂️

Reply:
If the solution seems tricky, I could throw some ideas:

In `TooltipStyleUtils/types.ts` we could add...

```ts
interface PlatfromTooltipStyles {
  rootWrapperTopOverride?: number, // On Web, use `shouldShowBelow` and compute an "override" of the default computation
  
}

interface PlatfromTooltipStyleUtil {
  getPlatfromTooltipStyles(whateverArgsAreNecessary): 
}
```

Create...
- `TooltipStyleUtils/platfromTooltipStyleUtil/index.ts
- `TooltipStyleUtils/platfromTooltipStyleUtil/index.native.ts

```ts
const platfromTooltipStyleUtil: PlatfromTooltipStyleUtil = {
  // implement, but differently on both platfroms
}
```

Usage...

```ts
const createTooltipStyleUtils: StyleUtilGenerator<GetTooltipStylesStyleUtil> = ({theme, styles}) => ({
    getTooltipStyles: ({
        tooltip,
        currentSize,
        windowWidth,
        xOffset,
        yOffset,
        tooltipTargetWidth,
        tooltipTargetHeight,
        maxWidth,
        tooltipContentWidth,
        tooltipWrapperHeight,
        manualShiftHorizontal = 0,
        manualShiftVertical = 0,
        shouldForceRenderingBelow = false,
        shouldForceRenderingLeft = false,
        wrapperStyle = {},
    }) => {
        const platformStyles = platfromTooltipStyleUtil.createStyles(somePassedArgs);
    
        const customWrapperStyle = StyleSheet.flatten(wrapperStyle);
        const tooltipVerticalPadding = spacing.pv1;

        // ...
```